src/arrayMethodSort.js

```diff
@@ -4,9 +4,42 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let changesCounter;
+
+    do {
+      changesCounter = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        if (compareFunction(this[i - 1], this[i]) >= 1) {
+          const previousElement = this[i - 1];
+
+          this[i - 1] = this[i];
+          this[i] = previousElement;
+          changesCounter++;
+        }
+      }
+    } while (changesCounter > 0);
+
+    return this;
   };
+
+  function compareAsStrings(a, b) {
+    const x = String(a);
+    const y = String(b);
+
+    if (x > y) {
+      return 1;
+    }
+
+    if (x < y) {
+      return -1;
+    }
+
+    if (x === y) {
+      return 0;
+    }
```

Comment:
Final `if` is redundant.
```suggestion
    if (x < y) {
      return -1;
    }

    return 0;
```